cabal-install/Distribution/Client/ProjectOrchestration.hs

```diff
@@ -608,7 +616,7 @@ reportBuildFailures verbosity plan buildOutcomes
 
        -- For all failures, print either a short summary (if we showed the
        -- build log) or all details
-       die $ unlines
+       notice verbosity $ unlines
```

Comment:
I don't think we have one, though IMO it makes more sense for `die` to print errors on stderr even with `-v0`.